internal/pkg/manifest/validate.go

```diff
@@ -523,6 +523,9 @@ func (s *SidecarConfig) Validate() error {
 			return fmt.Errorf(`validate "mount_points[%d]: %w`, ind, err)
 		}
 	}
+	if err := s.HealthCheck.Validate(); err != nil {
+		return fmt.Errorf(`validate "healthcheck": %w`, err)
+	}
```

Comment:
can we add a prefix test for this 🥺

Reply:
It's not possible yet boss, since the function doesn't return any error.

Reply:
I'll add it once I add more tests to the boilerplate.